content/swift/concepts/sets/terms/isSubset/isSubset.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: '.isSubset()'
+Description: 'Checks whether all elements of a set are present in another set.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Sets'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`isSubset()`** method returns `true` if all elements of a set are present in another set(passed as an argument). If not, it returns false.
```

Comment:
```suggestion
The **`isSubset()`** method returns `true` if all elements of a set are present in another set (passed as an argument), false otherwise.
```